secrets-provider/aws/README.md

```diff
@@ -64,7 +64,7 @@ pulumi up --yes
 Previewing update (aws-kms):
      Type                    Name                    Plan
  +   pulumi:pulumi:Stack     pulumi-aws-kms-aws-kms  create
- +   ├─ aws:s3:Bucket        bucket                  create
+ +   ├─ aws:s3:BucketV2      bucket                  create
  +   └─ aws:s3:BucketObject  secret                  create
```

Comment:
Shouldn't this also update to `BucketObjectV2` or is that resource still fine?

Reply:
I've avoided rewriting BucketObject into BucketObjectV2 intentionally. These currently share implementation but are distinct TF resources with identical schema. I'm hoping we can silently cut over to BucketObjectV2 implementation in pulumi-aws V7.

Reply:
Fair enough! I left similar comments on 1-2 other PRs, feel free to just resolve them